core-samples/README.md

```diff
@@ -2,6 +2,24 @@
 
 The samples in this directory demonstrate using [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) ES modules with various frameworks, module bundlers and build tools. `@arcgis/core` is installed using [npm](https://docs.npmjs.com/downloading-and-installing-packages-locally), and is intended for use in local builds that you host.
 
+## Download
+
+You can use these links to download all of the samples, or individual samples:
```

Comment:
@andygup we don't have a way yet to allow for downloading all samples, right?

```suggestion
You can use these links to download individual samples:
```

Reply:
Oops, I missed that bit of wording.